src/components/FilterPane/FilterPane.styles.js

```diff
@@ -131,11 +146,26 @@ export const StyledListSubheader = styled(ListSubheader)`
 export const StyledClickableArea = styled('div')`
   cursor: pointer;
   display: flex;
-  &:hover {
+  padding: 0.5rem;
+  ${hoverState(css`
     background-color: ${theme.color.tableRowHover};
-  }
+  `)}
 `
 
 export const StyledLabel = styled('label')`
   cursor: pointer;
+  margin: 0.2rem;
+`
+
+export const StyledCategoryContainer = styled('div')`
```

Comment:
When styling any standard html element (`div`, `span` etc) you can shorten the syntax from `styled('div')` to `styled.div`